src/DotNetLightning.Core/Channel/CommitmentsModule.fs

```diff
@@ -449,9 +449,9 @@ module ForceCloseFundsRecovery =
                     (lockTime.ToString())
                     (sequence.ToString())
 
-    let private tryGetObscuredCommitmentNumber (fundingOutPoint: OutPoint)
-                                               (transaction: Transaction)
-                                                   : Result<ObscuredCommitmentNumber, ValidateCommitmentTxError> = result {
+    let tryGetObscuredCommitmentNumber (fundingOutPoint: OutPoint)
```

Comment:
Why it has to be public? The reason is not written in commit msg.

Reply:
It's used in geewallet to identify commitment transactions that have been broadcast to the blockchain by the remote peer.

Reply:
Fair enough.